### PR TITLE
MPU9250: fix for edison build

### PR DESCRIPTION
**Pull request by AD-Aleksandrov**
Replacement for the older PR https://github.com/PX4/DriverFramework/pull/134

### CODE REVIEW

**Comment by julianoes** [comment]
@AD-Aleksandrov awesome, thank you.
And I am correct that we need no changes on the Firmware side now?

File: drivers/mpu9250/MPU9250.hpp [hunk 1]
```diff
@@ -232,7 +237,11 @@ class MPU9250: public ImuSensor
 		_last_temp_c(0.0f),
 		_temp_initialized(false),
 		_mag_enabled(mag_enabled),
+#if defined(__EDISON)
+		_packets_per_cycle_filtered(1.0f), // The FIFO is supposed to run at 1kHz and we sample at 250Hz.
```
Comment (staroselskii) [inline drivers/mpu9250/MPU9250.hpp]: Isn't it supposed to be ```4.0f```?

Comment (AD-Aleksandrov) [inline drivers/mpu9250/MPU9250.hpp]: Yes, you are right.

**Comment by AD-Aleksandrov** [comment]
I have now idea, I do not setup Firmware.

**Comment by julianoes** [comment]
@AD-Aleksandrov: Ok, I think it's correct now. Thanks a lot for adapting this many times.

**Pull request merged**